atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunnerImpl.java

```diff
@@ -71,6 +74,7 @@
  */
 public class SweepTaskRunnerImpl implements SweepTaskRunner {
     private static final Logger log = LoggerFactory.getLogger(SweepTaskRunnerImpl.class);
+    public static final int CELL_BATCH_SIZE = 1;
```

Comment:
should this be added to the args for run so the caller can control the cell batch size too?